packages/jellyfish-api-core/__tests__/category/masternode/getAnchorTeams.test.ts

```diff
@@ -0,0 +1,73 @@
+import { RegTestFoundationKeys } from '@defichain/jellyfish-network'
+import { TestingGroup } from '@defichain/jellyfish-testing'
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+
+describe('Masternode', () => {
+  const tGroup = TestingGroup.create(1)
+
+  beforeAll(async () => {
+    await tGroup.start()
+  })
+
+  afterAll(async () => {
+    await tGroup.stop()
+  })
+
+  it('should be empty when block is not 15', async () => {
+    for (let i = 0; i < 15; i++) {
+      const blockNumber = await tGroup.get(0).rpc.blockchain.getBlockCount()
+      expect(blockNumber).toBeLessThan(15)
```

Comment:
Blocks are not automatically generated. Hence `blockNumber`  remains `0`

Reply:
my bad. i missed out the generate(1)